master/getting-started/kubernetes/tutorials/stars-policy/index.md

```diff
@@ -31,7 +31,7 @@ kubectl get pods --all-namespaces --watch
 The management UI runs as a `NodePort` Service on Kubernetes, and shows the connectivity
 of the Services in this example.
 
-If you're running the vagrant cluster, you can view the UI by visiting `http://172.18.18.102:30002` in a browser.  For other clusters, accessing
+If you're running the vagrant cluster, you can view the UI by visiting `http://<vagrant-ip>:30002` in a browser.  For other clusters, accessing
```

Comment:
these tutorial (as of now) are independent of the setup, and depending on the vagrantfile config people can have different IPs. A lot of people follow coreOS multi-node vagrantfile or minikube, or any other k8s installation online and they're not going to have the same IPs